wikiwiki/PublicAccess.md

```diff
@@ -6,4 +6,9 @@
 
 ## Today's Opinions
 * Public users have access to all opinions served today via the Today's Opinion page
-* Opinions in sealed cases are available in the Today's Opinions list
+* Opinions in sealed cases are available in the Today's Opinions list and the public can access the PDF for a  opinions in sealed cases
```

Comment:
just fix the wee typo "for a  opinions..." => "for opinions..."

Reply:
Done